index.html

```diff
@@ -8,59 +8,73 @@
 </head>
 <body>
 	<header>
+		<!-- My name -->
 		<h1>
+			Caleb Mahala
 		</h1>
-		<!-- My name -->
 		<aside>
-			<a href="/"></a>
+			<!-- To call me -->
+			<a href="/">(717)-814-0959</a>
 			<!-- To email me -->
+			<a href="/">cgmahala171@stevenscollege.edu</a>
 		</aside>
 	</header>
 	<article>
+		<!-- Objectives -->
 		<section>
-			<p><h3></h3>
+			<p><h3>Objectives/Goals</h3>To obtain a position where I can use my current front end development training to help develop and create websites to further my experience in computer software engineering.
 			</p>
 		</section>
-		<!-- Objectives -->
+		<!-- Education and Skills -->
 		<section>
 			<div>
-				<p><h3></h3>
+				<h3>Education</h3>
+				<p><h4>Hempfield High School, Landisville PA </h4>
+					<h5>high school deploma</h5>
+					I graduated early from Hempfield High School to go to Thaddeus Stevens College of Technology where I am currently enrolled. I left with a 3.6 GPA and a SAT score of 1300.
+				</p>
```

Comment:
Give me your thoughts on the above p tag. should I have done this differently. I am trying to set it up like I did on my resume.dox page on html_content branch

Reply:
Change the first <p> tag to a div and end the div tag at the end of </h5>. Then create a new starting <p> tag and place it at the start of "I graduated early from Hempfield".